### DISCUSSION TITLE
4.0.0 Beta

### DISCUSSION THREAD

**Discussion by argaudreau** [mitre/caldera#2460] [Announcements]
# What's Changed

## Operations Page
  * Made more use of screen space at top of page
  * Adding a potential link now gives you the ability to edit the command before it's added
  * You can select fact values for all fact templates in a potential link, either ones from a fact source or ones collected from the operation.
  
## Training Plugin
  * UI has been refreshed to match the new UI in core CALDERA
  * Gameboard badge has been removed
  * New users should be able to complete User certificate in its entirety without issue

## Other
* API Docs are better documented
* Timestamps in sandcat are now UTC instead of local time
* More back end tests have been added 
* General bug squashing and improvements  

**Full Changelog**: https://github.com/mitre/caldera/compare/3.1.0...4.0.0-beta

## Contributors (since last release)
@ArtificialErmine, @clenk, @argaudreau, @iguannalin, @heatonk, @bleepbop, @mchan143, @christophert, @yee-jonathan, @blackwidow0616, @djlawren, @ddavila54, @CDJellen, @wbooth, @bernsteinj, @emmanvg, @cyber-arsenull, @uruwhy, @elegantmoose, @damionmounts, @zacharylc-mitre, @cmagone, @alexanderkent, ... and more!

## New Contributors
* @emmanvg made their first contribution in https://github.com/mitre/caldera/pull/2157
* @bleepbop made their first contribution in https://github.com/mitre/caldera/pull/2188
* @neptunia made their first contribution in https://github.com/mitre/caldera/pull/2224
* @Sloane4 made their first contribution in https://github.com/mitre/caldera/pull/2211
* @CDJellen made their first contribution in https://github.com/mitre/caldera/pull/2321
* @cyber-arsenull made their first contribution in https://github.com/mitre/caldera/pull/2346
* @heatonk made their first contribution in https://github.com/mitre/caldera/pull/2373
* @bernsteinj made their first contribution in https://github.com/mitre/caldera/pull/2411
* @BCHarrell made their first contribution in https://github.com/mitre/caldera/pull/2415
* @yee-jonathan made their first contribution in https://github.com/mitre/caldera/pull/2398
* @djlawren made their first contribution in https://github.com/mitre/caldera/pull/2404
* @damionmounts made their first contribution in https://github.com/mitre/caldera/pull/2424
* @zacharylc-mitre made their first contribution in https://github.com/mitre/caldera/pull/2418
* @cmagone made their first contribution in https://github.com/mitre/caldera/pull/2440

Thank you to all of the MANY builders of CALDERA, both in and out of GitHub! 🚀

<hr /><em>This discussion was created from the release <a href='https://github.com/mitre/caldera/releases/tag/4.0.0-beta'>4.0.0 Beta</a>.</em>

**Comment by h0tm1xx3** [comment]
Hi Caldera team,

I am currently working in the training. But some of the training solution guide is not updated. For instance in the Basic Operation... There is no "Hunter" in the adversary list. Is there any possible work around until the next available update? 
![No Hunter](https://user-images.githubusercontent.com/102936977/161466905-98260db3-03be-4f07-b45e-da3d71323b70.PNG)

**Reply by uruwhy** [reply]
> Hi Caldera team,
> 
> I am currently working in the training. But some of the training solution guide is not updated. For instance in the Basic Operation... There is no "Hunter" in the adversary list. Is there any possible work around until the next available update?

@h0tm1xx3 thank you for letting us know - this has been addressed in https://github.com/mitre/training/pull/149 . For now, use the "Ransack" adversary instead of Hunter. Let us know if you have any other questions!
